build.gradle.kts

```diff
@@ -29,14 +29,26 @@ dependencies {
     compile("io.ktor:ktor-server-netty:$ktorVersion")
     compile("io.prometheus:simpleclient_common:$prometheusVersion")
     compile("io.prometheus:simpleclient_hotspot:$prometheusVersion")
-    compile("no.nav.helse:streams:$navStreamsVersion")
     compile("com.github.kittinunf.fuel:fuel:$fuelVersion")
     compile("io.arrow-kt:arrow-core-data:$arrowVersion")
 
+
+    compile("org.apache.kafka:kafka-streams:2.1.1")
+    compile("no.nav.helse:streams:$navStreamsVersion")
+
+    compile("org.flywaydb:flyway-core:5.2.3")
+    compile("no.nav:vault-jdbc:1.3.1")
+    compile("org.postgresql:postgresql:42.2.5")
+
+    compile("commons-lang:commons-lang:2.6")
```

Comment:
commons bør tas bort siden det er tatt inn som avhengighet kun for én funksjon

Reply:
fixed